llvm/tools/llvm-exegesis/llvm-exegesis.cpp

```diff
@@ -455,6 +466,26 @@ static void runBenchmarkConfigurations(
               Measurement.PerSnippetValue, NewMeasurement.PerSnippetValue);
         }
       }
+    } else if (RepetitionMode ==
```

Comment:
This shared a lot of code with the `if (RepetitionMode == Benchmark::RepetitionModeE::AggregateMin) {` above.

This could probably be:

```
if (RepetitionMode == Benchmark::RepetitionModeE::AggregateMin) {
  Result = AggregateResults(AllResults, []() { ... });
} else {
  Result = AggregateResults(AllResults, []() { ... });
}
```

Reply:
`AggregateResults` and the two agrgegator implementations could be moved to a separate file with unit tests :)

Reply:
Moved them to `ResultAggregator.cpp` and added unit tests.